src/portal/src/i18n/lang/ja-jp-lang.json

```diff
@@ -0,0 +1,1594 @@
+{
+    "APP_TITLE": {
+        "VMW_HARBOR": "Harbor",
+        "HARBOR": "Harbor",
+        "VIC": "vSphere Integrated Containers",
+        "MGMT": "Management",
+        "REG": "Registry",
+        "HARBOR_SWAGGER": "Harbor Swagger",
+        "THEME_DARK_TEXT": "ダークモード",
+        "THEME_LIGHT_TEXT": "ライトモード"
+    },
+    "SIGN_IN": {
+        "REMEMBER": "ユーザー情報の保存",
+        "INVALID_MSG": "ユーザー名またはパスワードが無効です。",
+        "FORGOT_PWD": "パスワードを忘れた場合",
+        "HEADER_LINK": "ログイン",
+        "CORE_SERVICE_NOT_AVAILABLE": "コア サービスを使用できません。"
+    },
+    "SIGN_UP": {
+        "TITLE": "登録"
+    },
+    "BUTTON": {
+        "STOP": "停止",
+        "CANCEL": "キャンセル",
+        "OK": "OK",
+        "DELETE": "削除",
+        "LOG_IN": "ログイン",
+        "LOG_IN_OIDC": "OIDC プロバイダ経由でログイン",
+        "SIGN_UP_LINK": "アカウントを登録する",
+        "SIGN_UP": "登録",
+        "CONFIRM": "確認",
+        "SEND": "送信",
+        "SAVE": "保存",
+        "TEST_MAIL": "メールサーバーをテスト",
+        "CLOSE": "閉じる",
+        "TEST_LDAP": "LDAP サーバーをテスト",
+        "TEST_OIDC": "OIDC サーバーをテスト",
+        "MORE_INFO": "詳細情報...",
+        "YES": "はい",
+        "NO": "いいえ",
+        "NEGATIVE": "NEGATIVE",
+        "COPY": "コピー",
+        "EDIT": "編集",
+        "SWITCH": "切り替え",
+        "REPLICATE": "複製",
+        "ACTIONS": "アクション",
+        "BROWSE": "閲覧",
+        "UPLOAD": "アップロード",
+        "NO_FILE": "ファイルが選択されていません",
+        "ADD": "追加",
+        "RUN": "RUN",
+        "CONTINUE": "継続",
+        "ENABLE": "有効",
+        "DISABLE": "無効"
+    },
+    "BATCH": {
+        "DELETED_SUCCESS": "正常に削除されました",
+        "DELETED_FAILURE": "削除が失敗しました",
+        "SWITCH_SUCCESS": "切り替えが正常に完了しました",
+        "SWITCH_FAILURE": "切り替えに失敗しました",
+        "REPLICATE_SUCCESS": "複製が正常に開始しました",
+        "REPLICATE_FAILURE": "複製の開始に失敗しました",
+        "STOP_SUCCESS": "停止が正常に完了しました",
+        "STOP_FAILURE": "停止の実行に失敗しました",
+        "TIME_OUT": "Gateway time-out"
+    },
+    "TOOLTIP": {
+        "NAME_FILTER": "リソース名をフィルターします。全リソース名を指定するには空のままにするか、'**'を指定します。'library/**'と指定すると'library'配下のリソースのみがフィルターされます。詳しくは、ユーザーガイドをご覧ください。",
```

Comment:
Would you please use square brackets 「」for ' ' to align with VMware styleguide? For example, "「**」を指定します。" No space is required before and after square brackets.
Also, would you please add a space between Katakana compound words, like ユーザー ガイド. If it would cause too much efforts to apply it globally, please ignore.